extensions/adobe/experience/adcloud/campaign.schema.json

```diff
@@ -14,46 +14,46 @@
     "campaign": {
       "properties": {
         "xdm:accountId": {
-          "title": "Account ID",
+          "title": "Account Identifier",
           "type": "string",
-          "description": "Unique identifier for the account this event is associated with."
+          "description": "Identifier for the account."
```

Comment:
@sathyz/ @richaG99  : curious about why the `Unique` qualification was removed ?

Reply:
@skannan-adobe the commit came from chris.